### PR TITLE
Push log to elk

### PR DESCRIPTION
**Pull request by hieunguyen23**
Push log to ELK

### CODE REVIEW

**Comment by hieunguyen23** [comment]
Push log to ELK

File: zimbra_all/opt/push-logs-to-ELK/60-output.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# output {
+#   if [type] == "rsyslog" {
+#     elasticsearch {
+#       hosts => [ "elasticsearch_private_ip:9200" ]
+#     }
+#   }
+# }
+
+*.*                         @logstash_ip:10514;json-template
```
Comment (tuanta) [inline zimbra_all/opt/push-logs-to-ELK/60-output.conf:35]: This should be a specific IP (of Logstash), isn't it?
So how to pass that value to? (maybe through a parameter on "docker run"?)

File: zimbra_all/opt/build.sh [hunk 2]
```diff
@@ -20,6 +20,10 @@
 if [ -e /opt/zimbra-install/install-autoKeys ]
 then ## Zimbra NOT installed yet.
 
+## Push-logs-to-ELK
+cp /opt/push-logs-to-ELK/* /etc/rsyslog.d/
```
Comment (tuanta) [inline zimbra_all/opt/build.sh:24]: How about clean up these files after one time setup?

**Review by tuanta — CHANGES_REQUESTED** [review]
Check these comments, improve your code then pull request back. Thanks.

**Comment by tuanta** [comment]
@hieunguyen23 không muốn merge nữa à :)

**Pull request closed without merging**